domain/consensus/processes/blockbuilder/test_block_builder.go

```diff
@@ -206,21 +231,48 @@ func (bb *testBlockBuilder) BuildUTXOInvalidBlock(parentHashes []*externalapi.Do
 		return nil, err
 	}
 
-	// We use genesis transactions so we'll have something to build merkle root and coinbase with
+	bits, err := bb.difficultyManager.RequiredDifficulty(stagingArea, tempBlockHash)
+	if err != nil {
+		return nil, err
+	}
+
+	_, err = bb.difficultyManager.StageDAADataAndReturnRequiredDifficulty(stagingArea, tempBlockHash, false)
+	if err != nil {
+		return nil, err
+	}
+	daaScore, err := bb.daaBlocksStore.DAAScore(bb.databaseContext, stagingArea, tempBlockHash)
+	if err != nil {
+		return nil, err
+	}
+
+	ghostdagData, err := bb.ghostdagDataStore.Get(bb.databaseContext, stagingArea, tempBlockHash, false)
+	if err != nil {
+		return nil, err
+	}
+	blueWork := ghostdagData.BlueWork()
+
+	// We use the genesis coinbase so that we'll have something to build merkle root and a new coinbase with
 	genesisTransactions := bb.testConsensus.DAGParams().GenesisBlock.Transactions
+	genesisCoinbase := genesisTransactions[transactionhelper.CoinbaseTransactionIndex].Clone()
+	binary.LittleEndian.PutUint64(genesisCoinbase.Payload[:8], ghostdagData.BlueScore())
```

Comment:
Two of the tests create UTXOInvalid blocks that do reach validateBlockBodyInContext. Using `ExpectedCoinbaseTransaction` would be complicated here, because it depends on a few things that don't exist for UTXOInvalid sub-DAGs